chugbot/meta/ajax.js

```diff
@@ -107,11 +107,11 @@ $(function() {
 			var sessionAndGroup = listName.split("||");
 			if (listElements.length < expectedChugCount) {
 			    errorCount++;
-			    chugCountError += "Only " + listElements.length + " chugim selected for " + sessionAndGroup[0] + ", " + sessionAndGroup[1] + " (you must choose " + expectedChugCount + ")<br>";
```

Comment:
It was a bit easier to just slightly change these messages instead of passing through the constants all the way from PHP to here.